src/shared/constants.h

```diff
@@ -184,6 +184,11 @@ static const unsigned DIV_B_NUM_ROBOTS = 6;
 static const int MAX_KICK_CONSTANT       = 850;
 static const double MAX_KICK_COEFFICIENT = 0.4;
 
+// Chip Distance to Pulse Width Safety Constraint Constants
+// TODO: UPDATE CONSTANTS ONCE MORE CHIP DATA IS TAKEN FOR ROBOTS
+static const uint32_t MAX_CHIP_PULSE_WIDTH = 7500;
```

Comment:
you need to `#include <cstdint>` or just use an int and `static_cast` later since you're already doing it